src/components/commons/bottomSheet/viewBottomSheet/ViewBottomSheet.tsx

```diff
@@ -0,0 +1,27 @@
+import * as S from "./ViewBottomSheetStyle";
+
+import BottomSheet from "../BottomSheet";
+import ContextBox from "@components/commons/contextBox/ContextBox";
+import { ReactNode } from "react";
+
+interface viewBottomSheetPropsTypes {
+  title?: string;
+  boxTitle?: string;
+  children?: ReactNode;
+}
+
+const ViewBottomSheet = ({ title, boxTitle, children, ...rest }: viewBottomSheetPropsTypes) => {
+  return (
+    <S.ViewBottomSheetWrapper>
+      <BottomSheet title={title}>
+        <ContextBox customPadding="1.6rem 1.6rem">
```

Comment:
p4) 그럼 여기서도 전달해야겠죠!?
```suggestion
        <ContextBox customPadding="1.6rem 1.6rem" {...rest}>
```